src/profile/helpers/getAddress.ts

```diff
@@ -5,9 +5,11 @@ import { MyProfileQuery } from '../../graphql/generatedTypes';
 export default function getAddress(data: MyProfileQuery, lang: string) {
   if (data.myProfile?.primaryAddress) {
     const address = data.myProfile.primaryAddress;
-    return `${address.address}, ${address.city} ${
-      address.postalCode
-    } \n ${countries.getName(address.countryCode || 'FI', lang)}`;
+    const country = countries.getName(address.countryCode || 'FI', lang);
+
+    return [address.address, address.city, address.postalCode, country].join(
+      ', '
+    );
```

Comment:
All of the fields are not required, filling only postalCode for example will result in `, , 12345, Finland`.

Reply:
Oh ok, the previous implementation also had this issue with missing address/city so I ignored it. We can just filter `undefined` values out in which case the result does not have extra commas. It may result in weird addresses like "Finland" though. Would that be an issue 🤔

Reply:
True, it was an issue before. I just thought that if we are making improvements to UI, it would be nice to fix this as well :)